bigbone-rx/src/main/kotlin/social/bigbone/rx/admin/RxAdminAccountMethods.kt

```diff
@@ -0,0 +1,184 @@
+package social.bigbone.rx.admin
+
+import io.reactivex.rxjava3.core.Completable
+import io.reactivex.rxjava3.core.Single
+import social.bigbone.MastodonClient
+import social.bigbone.api.Pageable
+import social.bigbone.api.Range
+import social.bigbone.api.entity.admin.AccountOrigin
+import social.bigbone.api.entity.admin.AccountStatus
+import social.bigbone.api.entity.admin.ActionAgainstAccount
+import social.bigbone.api.entity.admin.AdminAccount
+import social.bigbone.api.method.admin.AdminAccountMethods
+
+/**
+ * Reactive implementation of [AdminAccountMethods].
+ *
+ * Perform moderation actions with accounts.
+ *
+ * @see <a href="https://docs.joinmastodon.org/methods/admin/accounts/">Mastodon admin/accounts API methods</a>
+ */
+class RxAdminAccountMethods(client: MastodonClient) {
+
+    private val adminAccountMethods = AdminAccountMethods(client)
+
+    /**
+     * View all accounts, optionally matching certain criteria for filtering, up to 100 at a time.
+     *
+     * @param range optional Range for the pageable return value
+     * @param origin Filter for [AccountOrigin.Local] or [AccountOrigin.Remote]
+     * @param status Filter for [AccountStatus] accounts
+     * @param permissions Filter for accounts with <code>staff</code> permissions (users that can manage reports)
+     * @param roleIds Filter for users with these roles
+     * @param invitedById Lookup users invited by the account with this ID
+     * @param username Search for the given username
+     * @param displayName Search for the given display name
+     * @param byDomain Filter by the given domain
+     * @param emailAddress Lookup a user with this email
+     * @param ipAddress Lookup users with this IP address
+     *
+     * @see <a href="https://docs.joinmastodon.org/methods/admin/accounts/#v2">Mastodon API documentation: admin/accounts/#v2</a>
+     */
+    @JvmOverloads
+    fun viewAccounts(
+        range: Range = Range(),
```

Comment:
I’ve moved `range` way up here as all other parameters are optional and thus Java users would have to use the variant with all parameters even if they only wanted to use the `range` parameter.